roles/device_roles/etherip/templates/base.cmd.j2

```diff
@@ -0,0 +1,17 @@
+
+dbLoadDatabase("{{ deploy_ioc_template_root_path }}/dbd/{{ deploy_ioc_executable }}.dbd")
+{{ deploy_ioc_executable }}_registerRecordDeviceDriver(pdbbase)
+
+# etherip specific commands
+## Load PLC driver
+EIP_buffer_limit(492)
+drvEtherIP_init()
+drvEtherIP_define_PLC("$(PLC)","$(PLC_IP)",0)
```

Comment:
We should add `PLC` and `PLC_IP` as required env vars in the schema.

Reply:
Ah good catch, thank you.